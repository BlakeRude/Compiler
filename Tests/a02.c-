// C-S21
main()
{
     int x;

     output(982);
     outnl();
}

