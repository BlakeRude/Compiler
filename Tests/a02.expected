WARNING(4): The variable 'x' seems not to be used.
Number of warnings: 1
Number of errors: 0
------------------------------------
Loading file: a02.tm
982
Bye.
